package.json

```diff
@@ -12,7 +12,7 @@
   ],
   "scripts": {
     "build": "yarn workspace env build && yarn workspace engine build && yarn workspace server build && yarn workspace client build",
-    "start": "node ./dist/server/index.js --env=production"
+    "start": "node ./dist/server/index.js --inspect --env=production"
```

Comment:
I don't think this will break anything? but it will make the production builds for snakeymouse.rawb.tv less efficient. Should probably revert